vue/client/base.vue

```diff
@@ -42,18 +42,3 @@ while (radius < Math.min(wrapperWidth, wrapperHeight) / 2) {
 
 tiles.value = newTiles
 </script>
-
-<style scoped>
```

Comment:
We shouldn't just remove the style block altogether. No one typically writes code like this

Reply:
Also about the whole idiomatic thing... I would only add 'scoped' to Vue if I really need it, so I would leave it out. But the <style> block should remain, because yes, that's where most of your styles would end up for a typical Vue component (or project).